### ISSUE TITLE
How to update version (stuck on 7-36)

### ISSUE THREAD

**Issue by Barraguesh** [flathub/com.valvesoftware.Steam.CompatibilityTool.Proton-GE#145]
I just discovered I was on 7-36, thought that being installed over Flatpak, it would update automatically. Removing the extension and reinstalling installed me the current latest (7-53). Is this the correct way of updating it, or should it happen automatically?

I think the README should address it, as I haven't been able to find any info on it, apart from the Flatpak apps being automatically updated by GNOME Software.

**Comment by Lctrs** [comment]
Yeah a flatpak update should have updated it automatically. I think maybe you messed up with your flatpak remotes somehow.

Won't had a note about it in the README as there is nothing specific to this flatpak app and this is rather generic to how flatpak works.

**Comment by Lctrs** [comment]
Then I don't know what happened but a flatpak update should have proposed you the updates.

Once again, this is pretty standard to all flatpaks. Unless there is something specific to this app, I won't add a note about it in the README.
